testnet/stacks-node/src/burnchains/mod.rs

```diff
@@ -19,13 +19,25 @@ use super::operations::BurnchainOpSigner;
 pub enum Error {
     CoordinatorClosed,
     IndexerError(burnchains::Error),
+    BurnchainError,
+    MaxFeeRateExceeded,
+    IdenticalOperation,
+    NoUtxos,
```

Comment:
UTXO is all caps

Reply:
Addressed in https://github.com/stacks-network/stacks-core/commit/ca885d07e101fd7bbe8db0c4b6ca62cc23dc1033